dts/binding-template.yaml

```diff
@@ -213,6 +214,12 @@ on-bus: <string describing bus type, e.g. "i2c">
 # Note that it only makes sense to combine 'default:' with 'required: false'.
 # Combining it with 'required: true' will raise an error.
 #
+# If a property has 'deprecated: True' is a means to convey both to the user
```

Comment:
The first sentence doesn't parse.  Maybe:
> A property with 'deprecated: True' indicates to both the user and the tooling that....

Reply:
Changed.